src/Model/Domain/TaskerPool/TaskerPoolRepository.php

```diff
@@ -19,9 +22,10 @@ public function __construct(\PDO $pdo)
     public function upsert(string $taskerPoolEntity)
     {
         $query = sprintf(
-            "INSERT INTO " . self::TABLE_NAME . "(`hostname`,`status`,`ts_available`) VALUES ('%s',%s,%s) ON DUPLICATE KEY UPDATE ts_available=UNIX_TIMESTAMP();",
+            "INSERT INTO %s (`hostname`,`status`,`ts_available`) VALUES ('%s',%s,%s) ON DUPLICATE KEY UPDATE ts_available=UNIX_TIMESTAMP();",
```

Comment:
There are two things here, decide what you are going to use: `UNIX_TIMESTAMP()` or `time()`?
Also look inside the `getAvailableHostnames()`.